index.html

```diff
@@ -33,16 +33,14 @@ <h2>Introduction</h2>
     <h2><a href="#install-upgrade">Install/Upgrade <span class="inline-code">cabal</span></a></h2>
 
     <p>
-      To install the <span class="inline-code">cabal</span> executable you can
-      use <a href="https://gitlab.haskell.org/haskell/ghcup-hs">ghcup</a> (if you're using
-      Linux), the <a href="https://www.haskell.org/platform/">Haskell
-      Platform</a>, install the <span class="inline-code">cabal-install</span>
-      package from your distributions package manager (if using Linux or Mac),
-      or download the source or prebuilt binary from
+      The recommended method to install the <span class="inline-code">cabal</span> executable is to use <a href="https://www.haskell.org/ghcup/">ghcup</a>, which can manage multiple versions of <span class="inline-code">cabal</span> on Linux, Mac and Windows.
+      Alternatively, you can install <span class="inline-code">cabal</span> using your distributions package manager (if you are using Linux or Mac), or download the source or prebuilt binary from
       the <a href="download.html">Download</a> page.
+    </p>
 
     <p>
-      If you already have the <span class="inline-code">cabal</span> executable you can upgrade it by running:
+      If you have installed <span class="inline-code">cabal</span> using <a href="https://www.haskell.org/ghcup/">ghcup</a>, then you can also upgrade it using <span class="inline-code">ghcup</span>.
+      If you installed the <span class="inline-code">cabal</span> executable using a different method, you can upgrade it by running:
```

Comment:
This should mention that the cabal `installdir` needs to be in the `PATH` and come before the path where cabal was installed to previously.
Otherwise, the old cabal might prevail over the newly installed one...

Reply:
Done in cf9a680